minimum-constraints.txt

```diff
@@ -59,7 +59,9 @@ mock==2.0.0
 ordereddict==1.1
 ply==3.10
 PyYAML==5.1
-six==1.10.0
+six==1.10.0; python_version <= '3.7'
```

Comment:
Boy am I getting picky.  These kinds of comparisons are easier to understand and review of version number is same for both tests and use something like < rather than <= for compare.

Two occurances

Reply:
DONE